### ISSUE TITLE
Canvas Assignments for Project

### ISSUE THREAD

**Issue by jayfo** [uwcse510/web-cse510-wi18#9]
After I merge #7, the project page will show 7 red links.  Those same links also appear on some other pages.  Each corresponds to an assignment we need to create in Canvas.  Dates for each are also in the page.  Set them as due 11:59pm the night before, but ensure Canvas is not set to close.  We'll take "late" submissions that evening without complaint.

So the TODO here are:

- create the Canvas assignments
- fix the links to them in projects.md (in the YAML header, then removing the ` causing the red links)
- fix the corresponding links in calendar.md (just required adding in the YAML header)

**Comment by jesscs** [comment]
Some notes about the assignments I created:

- I made things due at 4am again; is that still what you want?

- I didn't bother with any "available after" stuff (which I think limits when they can submit stuff) because I'm not too worried about people submitting early, but I can if you want me to? 

- I made the final report 100 points, because that's what mine was worth in 2016.  Everything else is 0 points.  Is that right? 

- I made "end of day Monday, March 12, 2018." 11:59PM; let me know if that should change!
